.github/workflows/CI.yml

```diff
@@ -743,11 +743,13 @@ jobs:
 
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
+        with:
+          cache-disabled: true
```

Comment:
Why?

Reply:
added this changed from the work around - https://github.com/gradle/gradle-build-action/security/advisories/GHSA-h3qr-39j9-4r5v
`If the Gradle project does not opt-in to using the configuration cache, then it is not vulnerable.`

Reply:
You can re-enable the cache. We do not need to disable it globally for that alert.